packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app/configure-packages/react-native/configuremid.md

```diff
@@ -0,0 +1,12 @@
+Call `createConfig` function to create a configured client on the native modules. Resolves true if successfully configures a client. Note: `requireHardwareBackedKeyStore` is a configurable setting only on Android devices. If you're testing on Android emulators, set this field to false
```

Comment:
* "if _it_ successfully configures"
* "this field to `false`."

Notes should be formatted on new lines:

```
> **Note:**
```